New website v1

Dominik-MCC opened the pull request:
First, static version of our new Website. Social Media Posts are prepared, all placeholders are removed. Ready for review and publishing.

Dominik-MCC committed:
Restructuring and adding content

Dominik-MCC committed:
Adding Favicon and Events

MalteRei commented:
When I open index.html the Together headline and arrows are black>
![image](https://user-images.githubusercontent.com/37665562/97140762-a30f0500-175d-11eb-875a-3c6dba886b0a.png)
![image](https://user-images.githubusercontent.com/37665562/97140785-ac986d00-175d-11eb-883a-2bf9f3ad1f10.png)

MalteRei commented:
Is twitter intentionally twice in the footer? Once in the channels section and again under get in touch.

Commit:
Fix titel and urls

The footer email titel now is not about twitter.
The links in index.ts point to the right html files.

Commit:
Add build script to start website with npm run-script build

Commit:
Fix link on how to join to point to application

Commit:
Change stuff to projects

Commit:
Change url from /events to /calendar

/events in node seems to mess up the static files loading

Commit:
Fix static urls in Legacy event pages

Commit:
Update lecturesathome.html

Check back in September seems a bit out-dated

Dominik-MCC committed:
Updated ID

Dominik-MCC committed:
Updated selector for privacy
